Emby.Server.Implementations/Collections/CollectionManager.cs

```diff
@@ -232,10 +232,7 @@ private async Task AddToCollectionAsync(Guid collectionId, IEnumerable<Guid> ids
 
             if (list.Count > 0)
             {
-                var newList = collection.LinkedChildren.ToList();
-                newList.AddRange(list);
-                collection.LinkedChildren = newList.ToArray();
-
+                collection.LinkedChildren = collection.LinkedChildren.Concat(list).ToArray();
```

Comment:
Not much of an improvement. The sizes of both collections are known, so a better way would be to allocate a fixed size array with `collection.LinkedChildren.Length + list.Count` elements.